skimage/measure/fit.py

```diff
@@ -404,17 +520,37 @@ class EllipseModel(BaseModel):
 
     >>> xy = EllipseModel().predict_xy(np.linspace(0, 2 * np.pi, 25),
     ...                                params=(10, 15, 8, 4, np.deg2rad(30)))
-    >>> ellipse = EllipseModel()
-    >>> ellipse.estimate(xy)
-    True
+    >>> ellipse = EllipseModel.from_estimate(xy)
     >>> np.round(ellipse.params, 2)
     array([10.  , 15.  ,  8.  ,  4.  ,  0.52])
     >>> np.round(abs(ellipse.residuals(xy)), 5)
     array([0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0.,
            0., 0., 0., 0., 0., 0., 0., 0.])
+
+    The estimation can fail when — for example — all the input or output
+    points are the same.  If this happens, you will get an ellipse model for
+    which ``bool(model)`` is ``False``:
+
+    >>> # bool on a successful model (as above) gives True:
+    >>> bool(ellipse)
```

Comment:
I think we should use the `if model` pattern consistently everywhere, and not encourage `bool(model)`.

Reply:
Yup, sorry, missed that one - changed.